afs-core/src/test/java/com/powsybl/afs/AfsBaseTest.java

```diff
@@ -241,7 +241,7 @@ public void archiveAndUnarchiveTestWithZip() throws IOException {
         Path rootDir = fileSystem.getPath("/root");
         Files.createDirectory(rootDir);
         Files.createDirectory(rootDir.resolve("test"));
-        dir1.archive(rootDir.resolve("test"), true, false, new HashMap<>());
+        dir1.archive(rootDir.resolve("test"), true, false, new HashMap<>(), new HashMap<>());
```

Comment:
should use the previous method with no extra args (will use internally a default value)